.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,32 @@
+name: Documentation
```

Comment:
You should try running `make docs` on this repo on your local machine. See if it actually works.